tests/unit/modules/network/eos/test_eos_lag_interfaces.py

```diff
@@ -277,6 +277,32 @@ def test_eos_lag_interfaces_parsed(self):
         ]
         self.assertEqual(parsed_list, result["parsed"])
 
+    def test_eos_lag_interfaces_fallback(self):
+        set_module_args(
+            dict(
+                config=[
+                    dict(
+                        name="Port-Channel1",
+                        fallback=dict(mode="static", timeout=1),
+                        members=[
+                            dict(member="Ethernet1", mode="on"),
+                            dict(member="Ethernet2", mode="on"),
+                        ],
+                    )
+                ]
+            )
+        )
+        commands = [
+            "interface Ethernet1",
+            "channel-group 1 mode on",
+            "interface Ethernet2",
+            "channel-group 1 mode on",
+            "interface Port-Channel1",
+            "port-channel lacp fallback static",
+            "port-channel lacp fallback timeout 1",
+        ]
+        self.execute_module(changed=True, commands=commands)
+
```

Comment:
Please add a test case to check the idempotent state, when fallback parameter is used.